contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -282,17 +308,11 @@ abstract contract VaultHub is PausableUntilWithRoles {
     /// @dev permissionless if the vault's min reserve ratio is broken
     function forceRebalance(address _vault) external {
```

Comment:
:warning: it should still work for the undercollaterized vaults

Reply:
🙏  Subject of a separate PR, will do!